package.json

```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "start": "node index.js",
     "lint": "eslint .",
-    "test": "jest"
+    "lint:fix": "npm run lint -- --fix",
```

Comment:
Either use `npm run` approach in both scripts `lint:fix` and `test:watch`/`test:coverage` or simply call the original command _i.e. eslint or jest (preferred)_ in both cases but code must be consistent along the whole file.

Reply:
Nvm, commit the minor fix myself to be able to merge the PR.
Thanks for the contribution!